docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+
+Note that the Mesosphere DCOS comes in two versions: a **Community Edition** (CE), which is available (today) for cloud-only deployments, and the **Enterprise Edition** (EE), which can be used in a cloud setup, on-premise or in a hybrid setup. Some of the above listed features, incl. premium support, extended security and monitoring are exclusively available through the EE.
+
+## 4. Components
+
+### 4.1  Prerequisites
+
+The only prerequisite to use Kubernetes on DCOS is one of the following environments (physical and/or VM):
+
+1. Instances in a public-cloud, IaaS infrastructure such as [Google Compute](http://cloud.google.com/), AWS [EC2](http://aws.amazon.com/ec2/), and Microsoft [Azure](https://azure.microsoft.com/).
+1. An on-premise cluster of machines.
+1. A combination of 1. and 2., above.
+
+In **any case** above the physical or virtual machines **must not** be pre-installed with a local operating system—such as from the GNU/Linux or Windows family—as during the setup phase Mesosphere DCOS will install [CoreOS](https://coreos.com/). In future, other GNU/Linux flavors will be supported.
+
+### 4.2 Bill of Materials
+
+Core:
+
+* Cloud provisioning (AWS, Azure, GCE)
+* CoreOS
+* Mesos
+* Zookeeper, Exhibitor
+* Mesos-DNS
+* DCOS Distribution Package
+* DCOS Package Repository
+* DCOS CLI
+* DCOS GUI
+
+Included Services:
+
+* Kubernetes
+* Marathon
+* Chronos
+
+Installable Services:
+
+* HDFS
+* Kafka
+* Jenkins
+* Spark
+* Cassandra
+* Hadoop
+* YARN (via Myriad)
+* and many others
+
+## 5. Example Deployment
+
+In order to try out the example deployment described in the following, you first want to visit [mesosphere.com/install](http://mesosphere.com/install) and set up a cluster of machines with the Mesosphere DCOS installed.
+
+*Note that in the example deployment below you can use the native Kubernetes CLI commands and the respective DCOS sub-commands interchangeably. This means that the following two commands are semantically equivalent:* 
+
+```bash
+$ bin/kubectl get pods
+```
+
+and
+
+```bash
+$ dcos k8s get pods
+```
+
+Install Kubernetes on DCOS:
+
+```bash
+$ dcos package install kubernetes
+```
+
+Now, start the kubernetes-mesos API server, controller manager, and scheduler on a Mesos master node:
+
+```bash
+$ ./bin/km apiserver \
+  --address=${servicehost} \
+  --mesos_master=${mesos_master} \
+  --etcd_servers=http://${servicehost}:4001 \
+  --portal_net=10.10.10.0/24 \
+  --port=8888 \
+  --cloud_provider=mesos \
+  --v=1 >apiserver.log 2>&1 &
+
+$ ./bin/km controller-manager \
+  --master=$servicehost:8888 \
+  --mesos_master=${mesos_master} \
+  --v=1 >controller.log 2>&1 &
+
+$ ./bin/km scheduler \
+  --address=${servicehost} \
+  --mesos_master=${mesos_master} \
+  --etcd_servers=http://${servicehost}:4001 \
+  --mesos_user=root \
+  --api_servers=$servicehost:8888 \
+  --v=2 >scheduler.log 2>&1 &
+```
+
+Also on the master node, start up a proxy instance to act as a public-facing service router:
```

Comment:
I've been working on the dcos packaging for kubernetes-mesos. Couple of points here:
* users will not be spinning up binaries manually like in the getting started guide. instead once they do `dcos package install kubernetes` they'll basically be done.
* in the getting started guide the proxy is started on the master in order to act as a makeshift service router for the purposes of exposing the web UI. dcos is still working through some design issues regarding what frontend service routing will look like. however, it's not likely that the proxy will be abused this way in the dcos incarnation of the k8sm framework.
* since binaries are not spun up manually, the `disown` step a little further down is also unnecessary
* the kubectl invocations might end up looking more like `dcos k8s`, for example: `dcos k8s get pods`